Project/Application/AppFlowCoordinator.swift

```diff
@@ -43,13 +43,12 @@ extension AppFlowCoordinator: HelloWorldViewControllerDelegate {
 }
 
 extension AppFlowCoordinator: MainViewControllerDelegate {
-    func searchButtonAction() {
-
+    func selectedRow(with: Repository) {
+        setTabBarAsRoot()
```

Comment:
maybe we should also pass repository object to that function as we know that tab bar will need it?